### DISCUSSION TITLE
Detect if condition is true

### DISCUSSION THREAD

**Discussion by olirege** [ultralytics/yolov5#6034] [Q&A]
Hey guys! fun problem coming up:

Objects need to be detected only if a sensors state is true.

Objects are moving on a conveyor belt and trigger a photoelectric sensor when they're under the camera. 

How would this work?

**Comment by olirege** [comment]
I think my best bet would be to load the model from torch hub and feed the screenshots when they're available rather then using detect.py

**Reply by glenn-jocher** [reply]
Sounds good!

**Reply by olirege** [reply]
Any way to avoid downloading the Arial.fft? I'm trying to run this offline.

Cheers,
Oli

**Reply by olirege** [reply]
Oh, I'll just change the dockerfile

**Reply by glenn-jocher** [reply]
@olirege yes Dockerfile includes Arial.ttf preinstalled. Many systems already have this installed as well. If YOLOv5 is attempting to download it's because it can't be found locally.
